dockerfile_parse/parser.py

```diff
@@ -66,11 +66,13 @@ def __setitem__(self, key, value):
 
 
 class DockerfileParser(object):
-    def __init__(self, path=None, cache_content=False, env_replace=True):
+    def __init__(self, path=None, cache_content=False, env_replace=True, parent_env={}):
         """
         Initialize path to Dockerfile
         :param path: path to (directory with) Dockerfile
         :param cache_content: cache Dockerfile content inside DockerfileParser
+
```

Comment:
Caught me. Should say ':param env_replace: bool, whether to perform env var substitution'.